fossology/FOSSologyUI
# feat(UI): added manage group-users page

# Description

Added Manage Group Users Page.

### Changes

1. Added a new file [src/pages/Admin/Group/Manage/index.jsx]
3. Introduced a new route in `src/Routes.jsx`
4. Introduced new API callers for the UI integration.

## How to test

1. Go to the project's main directory.
2. run command `npm start`
3. Visit the page at `http://localhost:PORT/admin/group/manage` 
4. You will see a list of group members for the default selected group.
6. You can change the permission of any use in that group.

https://user-images.githubusercontent.com/66276301/183613559-6a682fe1-303c-46a1-83f2-d710092a3606.mp4

## Related Issue 
Closing #242

cc: @shaheemazmalmmd @GMishx
